src/cdk-experimental/popover-edit/edit-event-dispatcher.ts

```diff
@@ -6,16 +6,41 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {Injectable} from '@angular/core';
-import {Observable, Subject, timer} from 'rxjs';
-import {audit, distinctUntilChanged, filter, map, share} from 'rxjs/operators';
+import {Injectable, NgZone} from '@angular/core';
+import {combineLatest, MonoTypeOperatorFunction, Observable, pipe, Subject} from 'rxjs';
+import {
+  audit,
+  auditTime,
+  debounceTime,
+  distinctUntilChanged,
+  filter,
+  map,
+  share,
+  skip,
+  startWith,
+} from 'rxjs/operators';
 
 import {CELL_SELECTOR, ROW_SELECTOR} from './constants';
 import {closest} from './polyfill';
 import {EditRef} from './edit-ref';
 
-/** The delay between mouse out events and hiding hover content. */
-const DEFAULT_MOUSE_OUT_DELAY_MS = 30;
+/** The delay applied to mouse events before hiding or showing hover content. */
+const MOUSE_EVENT_DELAY_MS = 40;
+
+/** The delay for reacting to focus/blur changes. */
+const FOCUS_DELAY = 0;
+
+/**
+ * The possible states for hover content:
+ * OFF - Not rendered.
+ * FOCUSABLE - Rendered in the dom and stylyed for its contents to be focusable but invisible.
+ * ON - Rendered and fully visible.
+ */
+export const enum HoverContentState {
```

Comment:
Does this need to be exported outside of the package? If not we can omit it from `public-api.ts`

Reply:
At the very least, it's needed in material-experimental/popover-edit.

Would you prefer it be exported from cdk-experimental with a leading underscore?

Reply:
It's fine if it's necessary in the material implementation